### PR TITLE
Upgrade the Gradle wrapper to 8.7, the Gradle plugin to 0.7.0, and the embulk-spi to v0.11

### PR DESCRIPTION
**Pull request by hiroyuki-sato**
There are four commits

* Upgrade the Gradle wrapper to 8.7
* Use LICENSE file instead of COPYING
* Upgrade the Gradle plugin to 0.7.0
* Use embulk-spi v0.11

### CODE REVIEW

File: src/main/java/org/embulk/output/CommandFileOutputPlugin.java
Comment (hiroyuki-sato) [inline src/main/java/org/embulk/output/CommandFileOutputPlugin.java:1]: Should I fix this warning? I didn't know how to fix this.

```
embulk-output-command/src/main/java/org/embulk/output/CommandFileOutputPlugin.java:149: warning: [deprecation] array() in org.embulk.spi.Buffer has been deprecated
                currentProcess.write(buffer.array(), buffer.offset(), buffer.limit());
```

Comment (dmikurube) [reply src/main/java/org/embulk/output/CommandFileOutputPlugin.java:1]: For the time being, you can leave it.

File: build.gradle [hunk 1]
```diff
@@ -1,58 +1,169 @@
...
+gemPush {
+    host = "https://rubygems.org"
+}
+
+jar {
+    metaInf {
+        from rootProject.file("LICENSE")
+    }
+}
+
```
Comment (dmikurube) [inline build.gradle:72]: `javadocJar`?

File: src/main/java/org/embulk/output/CommandFileOutputPlugin.java
Comment (hiroyuki-sato) [reply src/main/java/org/embulk/output/CommandFileOutputPlugin.java:1]: Thanks!

**Review by dmikurube — APPROVED** [review]
LGTM but one comment.

File: build.gradle (hunk 1 above)
Comment (hiroyuki-sato) [reply build.gradle:72]: Thanks fixed in 7f3bcedc83318a5886268b5a2ccf7ac3723d3d96

**Comment by hiroyuki-sato** [comment]
Thanks!

**Pull request merged**